snippets/AlkFejlHfTanulsagok/31/snippet.md

```diff
@@ -0,0 +1,172 @@
+# A jeti123 csapat tanulságai az Alkalmazásfejlesztés házi feladattal kapcsolatban.

```

Comment:
Minden snippet file lején van egy fejléc, azt ne hagyjátok le. Anélkül meg sem fog jelenni...
Abban kell majd egy codename: AlkFejlHF34
A könyvtárat is nevezzétek át 34-re, mert párhuzamosan többen 31-et választottatok.